Project.toml

```
@@ -41,7 +41,7 @@ AbstractTrees = "0.4"
 ArrayInterface = "7.8"
 Bijections = "0.1.2, 0.2"
 ChainRulesCore = "1"
-Combinatorics = "1.0"
+Combinatorics = "1.0.2"
```

```suggestion
Combinatorics = "<=1.0.2"
```

A compat entry of `"1.0.2"` means "any package version from 1.0.2 to <2.0.0". What we need is "any package version below 1.0.2 inclusive"